src/synchronization/global_tas_lock.hpp

```diff
@@ -11,6 +11,7 @@
 #include "../data_distribution/global_ptr.hpp"
 #include <chrono>
 #include <thread>
+#include <atomic>
```

Comment:
our coding style doesn't say anything about it, but should these includes be in alphabetic order?